kubeflow/core/centraldashboard.libsonnet

```diff
@@ -2,6 +2,9 @@
   // TODO(https://github.com/ksonnet/ksonnet/issues/222): Taking namespace as an argument is a work around for the fact that ksonnet
   // doesn't support automatically piping in the namespace from the environment to prototypes.
 
+  // TODO(https://github.com/kubeflow/kubeflow/issues/527): The central UI is currently using a personal docker image: swiftdiaries/centraldashboard:0.3
```

Comment:
In a separate PR please create a Makefile like:
https://github.com/kubeflow/kubeflow/blob/master/bootstrap/Makefile

To build and push the image. I will then run it in order to push the image to gcr.io/kubeflow-images-public.

Reply:
Yeah, sure thing. Will get it done.